etc/config/c++.amazon.properties

```diff
@@ -643,6 +643,33 @@ group.armclang32.licenseLink=https://github.com/llvm/llvm-project/blob/main/LICE
 group.armclang32.licensePreamble=The LLVM Project is under the Apache License v2.0 with LLVM Exceptions
 group.armclang32.compilerCategories=clang
 
+compiler.armv7-clang1810.exe=/opt/compiler-explorer/clang-18.1.0/bin/clang
+compiler.armv7-clang1810.semver=18.1.0
+compiler.armv7-clang1810.options=-target arm-linux-gnueabi --gcc-toolchain=/opt/compiler-explorer/arm/gcc-13.2.0/arm-unknown-linux-gnueabihf --sysroot=/opt/compiler-explorer/arm/gcc-13.2.0/arm-unknown-linux-gnueabihf/arm-unknown-linux-gnueabihf/sysroot
+compiler.armv7-clang1701.exe=/opt/compiler-explorer/clang-17.0.1/bin/clang
+compiler.armv7-clang1701.semver=17.0.1
+compiler.armv7-clang1701.options=-target arm-linux-gnueabi --gcc-toolchain=/opt/compiler-explorer/arm/gcc-13.2.0/arm-unknown-linux-gnueabihf --sysroot=/opt/compiler-explorer/arm/gcc-13.2.0/arm-unknown-linux-gnueabihf/arm-unknown-linux-gnueabihf/sysroot
+compiler.armv7-clang1600.exe=/opt/compiler-explorer/clang-16.0.0/bin/clang
+compiler.armv7-clang1600.semver=16.0.0
+compiler.armv7-clang1600.options=-target arm-linux-gnueabi --gcc-toolchain=/opt/compiler-explorer/arm/gcc-12.2.0/arm-unknown-linux-gnueabihf --sysroot=/opt/compiler-explorer/arm/gcc-12.2.0/arm-unknown-linux-gnueabihf/arm-unknown-linux-gnueabihf/sysroot
+compiler.armv7-clang1500.exe=/opt/compiler-explorer/clang-15.0.0/bin/clang
+compiler.armv7-clang1500.semver=15.0.0
+compiler.armv7-clang1500.options=-target arm-linux-gnueabi --gcc-toolchain=/opt/compiler-explorer/arm/gcc-12.2.0/arm-unknown-linux-gnueabihf --sysroot=/opt/compiler-explorer/arm/gcc-12.2.0/arm-unknown-linux-gnueabihf/arm-unknown-linux-gnueabihf/sysroot
+compiler.armv7-clang1400.exe=/opt/compiler-explorer/clang-14.0.0/bin/clang
+compiler.armv7-clang1400.semver=14.0.0
+compiler.armv7-clang1400.options=-target arm-linux-gnueabi --gcc-toolchain=/opt/compiler-explorer/arm/gcc-11.2.0/arm-unknown-linux-gnueabihf --sysroot=/opt/compiler-explorer/arm/gcc-11.2.0/arm-unknown-linux-gnueabihf/arm-unknown-linux-gnueabihf/sysroot
+compiler.armv7-clang1301.exe=/opt/compiler-explorer/clang-13.0.1/bin/clang
+compiler.armv7-clang1301.semver=13.0.1
+compiler.armv7-clang1301.options=-target arm-linux-gnueabi --gcc-toolchain=/opt/compiler-explorer/arm/gcc-11.2.0/arm-unknown-linux-gnueabihf --sysroot=/opt/compiler-explorer/arm/gcc-11.2.0/arm-unknown-linux-gnueabihf/arm-unknown-linux-gnueabihf/sysroot
+compiler.armv7-clang1300.exe=/opt/compiler-explorer/clang-13.0.0/bin/clang
+compiler.armv7-clang1300.semver=13.0.0
+compiler.armv7-clang1300.options=-target arm-linux-gnueabi --gcc-toolchain=/opt/compiler-explorer/arm/gcc-11.2.0/arm-unknown-linux-gnueabihf --sysroot=/opt/compiler-explorer/arm/gcc-11.2.0/arm-unknown-linux-gnueabihf/arm-unknown-linux-gnueabihf/sysroot
```

Comment:
Am I missunderstanding something, or shouldnt these be the clang++ variants?

Reply:
They should be yes, I've fixed that.